07.bitstream.semantics.md

```diff
@@ -1520,7 +1519,8 @@ reference. It is a requirement of bitstream conformance that whenever
 expectedFrameId[ i ] is calculated, the value matches
 RefFrameId[ ref_frame_idx[ i ] ]
 (this contains the value of current_frame_id at the time that the frame indexed
-by ref_frame_idx was stored).
+by ref_frame_idx was stored) and RefValid[ ref_frame_idx[ i ] ] is equal
+to 1.
 
```

Comment:
As above, I don't think this is necessary

Reply:
Instead I think it may be better to adjust the misleading note that suggests RefValid only applies when using frame ids (see suggested text under the issue tracker)